src/javascripts/ng-admin/Crud/routing.js

```diff
@@ -111,7 +115,7 @@ function routing($stateProvider) {
                                 ).map(entry => dataStore.addEntry(references[name].targetEntity().uniqueId + '_values', entry));
                             }
                         }],
-                        entries: ['dataStore', 'view', 'response', 'referenceEntries', function (dataStore, view, response, referenceEntries) {
+                        entries: ['dataStore', 'view', 'response', function (dataStore, view, response) {
```

Comment:
This one is on purpose, it forces the ordering of the function execution in the `resolve()` logic. Please revert this change.